src/test_utils.rs

```diff
@@ -34,16 +34,18 @@ pub fn get_file_content<P: AsRef<Path>>(path: P) -> Vec<String> {
     }
     vec
 }
-pub fn get_lines_content<P: AsRef<Path>>(path: P) -> BTreeSet<String> {
+pub fn get_lines_content<P: AsRef<Path>>(path: P) -> Vec<String> {
     let path = path.as_ref();
     let file = File::open(path).unwrap_or_else(|_| panic!("file {:?} not found", path));
     let reader = BufReader::new(file);
-    let mut set = BTreeSet::new();
-    for result_line in reader.lines() {
-        let line =
-            result_line.unwrap_or_else(|_| panic!("Cannot parse as a line in file {:?}", path));
-        set.insert(line);
-    }
+    let mut set: Vec<String> = reader
```

Comment:
Can you please rename the variable if you really are keeping a `Vec` since it's not a `set` anymore?